<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CAN Communication</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>351</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lableCanDataBitrate">
       <property name="text">
        <string>CAN Data Bitrate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxCanName">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>can0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>can1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>vcan0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>vcan1</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lableCanname">
       <property name="text">
        <string>CAN Interface name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxCanDataBitrate">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500 KB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 MB</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelCanFD">
       <property name="text">
        <string>Use CAN FD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radioButtonUseCanFD">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioBoxCustomConfig">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Custom configuration</string>
    </property>
   </widget>
   <widget class="QLabel" name="lableReceivePayload">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Payload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSend">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDataReceive">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>200</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Received</string>
    </property>
   </widget>
   <widget class="QLabel" name="lableSendPayload">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Payload</string>
    </property>
   </widget>
   <widget class="QLabel" name="lableReceiveMsID">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="lableSendMsID">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textSendIdMs">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textReceiveIdMs">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textSendPayload">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textReceivePayload">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>gridLayoutWidget</zorder>
   <zorder>radioBoxCustomConfig</zorder>
   <zorder>lableReceivePayload</zorder>
   <zorder>labelDataReceive</zorder>
   <zorder>lableSendPayload</zorder>
   <zorder>lableReceiveMsID</zorder>
   <zorder>lableSendMsID</zorder>
   <zorder>buttonSend</zorder>
   <zorder>textSendIdMs</zorder>
   <zorder>textReceiveIdMs</zorder>
   <zorder>textSendPayload</zorder>
   <zorder>textReceivePayload</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTho_t">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionauthor"/>
   </widget>
   <addaction name="menuTho_t"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionauthor">
   <property name="text">
    <string>author</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
